org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/StaticPosition.java

```diff
@@ -46,13 +46,20 @@ public int getIndex(String name) {
 		return 0;
 	}
 
+	@SuppressWarnings("sync-override")
 	@Override
 	public List<Collection<String>> getDimensionNames() {
 		return Collections.emptyList();
 	}
 
 	@Override
 	public int getScanRank() {
+		return 1;
+	}
+
+	@Override
+	public int getIndex(int dimension) {
+		// TODO: how do we get the index when the number of StaticPositions is > 1?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)